### PR TITLE
WFLY-11442 Remove unused dependencies from org.jboss.as.ejb3

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
https://issues.redhat.com/browse/WFLY-11442

### CODE REVIEW

**Comment by bstansberry** [comment]
@chengfang or @tadamski Please review. Please note my comment at https://issues.redhat.com/browse/WFLY-11442?focusedCommentId=14017402&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14017402 re the analysis behind this. What went into it can be described in comments on the JIRA.

File: feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/ejb3/main/module.xml [hunk 1]
```diff
@@ -87,7 +82,6 @@
         <module name="org.jboss.ironjacamar.api"/>
         <module name="org.jboss.jandex"/>
         <!-- For SubordinationManager for remote Xid Transaction propagation -->
-        <module name="org.jboss.jts"/>
```
Comment (bstansberry) [inline feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/ejb3/main/module.xml:90]: If this is removed the comment on the line above should be removed as well.

**Review by tadamski — APPROVED** [review]

**Review by pferraro — APPROVED** [review]
Clustering module removals look fine.

**Review by yersan — COMMENTED** [review]
I've also taken a look at this. The removals looks fine.

It is tricky module, there are lot of classes doing their own class loading, although as far as I've been able to check, all of those class loadings are done from the deployment JBoss Modules module, so, they should be fine.

**Pull request merged**